public/javascripts/Gallery/css/tags.css

```diff
@@ -1,4 +1,4 @@
-.gallery-tag {
+.gallery-tag-modal {
```

Comment:
What you did:
1. Copied the `gallery-tag` class CSS into a new `gallery-tag-modal` class
2. Made the tags on the cards use the `gallery-tag-modal` class
3. Changed the `gallery-tag` class to use ellipses

What I'd like to see
1. Make a new `gallery-tag-sidebar` class, including only the new `width: 85%;` and `overflow: hidden;` properties
2. Reuse the `gallery-tag` CSS for the new `gallery-tag-sidebar` class. This would mean changing the `.gallery-tag {` line to `.gallery-tag, .gallery-tag-sidebar {`.

Reply:
> Reuse the gallery-tag CSS for the new gallery-tag-sidebar class. This would mean changing the .gallery-tag { line to .gallery-tag, .gallery-tag-sidebar {
I'm not sure I understand what you mean. Are you saying that you want me to have `.gallery-tag {}` `.gallery-tag-sidebar {}` and `.gallery-tag, .gallery-tag-sidebar {}` ?

Reply:
Probably just `.gallery-tag-sidebar` and `.gallery-tag, .gallery-tag-sidebar {}` because I think that the `gallery-tag` class doesn't have anything that the `gallery-tag-sidebar` class wouldn't have.

Reply:
I think it's the other way around the `gallery-tag-sidebar` doesn't have anything `gallery-tag` class doesn't have (kinda, since they do have some overlap).

Reply:
Gotcha. Yeah I probably just have it backwards: I'm moving pretty fast with all the stuff that's been going on this week :sweat_smile: 

Do you think that there is a way to consolidate a decent amount of the CSS so that it isn't repeated in multiple places without too much trouble? If so, I think that that would be helpful!

I also think that conceptually, having the sort of "default" `gallery-tag` class makes the most sense for the tags that are on the actual cards instead of the tags in the sidebar. So that is my main motivation for the name change.